packages/firebase_admob/ios/Classes/FLTMobileAd.m

```diff
@@ -101,6 +101,71 @@ - (NSString *)description {
 }
 @end
 
+@implementation FLTMobileAdWithView
+- (UIView *)adView {
+  // We cause a crash if this method is not overriden by subclasses.
+  [self doesNotRecognizeSelector:_cmd];
```

Comment:
Nope. Plugin users should never see this, but the plugin's maintainer would need to know this. I left this note so that another plugin maintainer knows why we cause a crash in this method.

Reply:
sounds good